app/models/person.rb

```diff
@@ -424,6 +424,18 @@ def has_active_enrollment
     false
   end
 
+  def ethnicity
+    # Retrieve the raw 'ethnicity' value from the database
+    # Return an empty array if 'ethnicity' is nil, ensuring consistent array behavior
+    read_attribute(:ethnicity) || []
```

Comment:
One consideration here-- this will create a discrepancy between what the object returns and what a direct query of the database returns for the nil case.  I don't have a particular scenario in mind where that causes a problem, just noting the nuance in case it sparks any additional thought.